src/Transport.ts

```diff
@@ -274,4 +282,23 @@ export class Transport {
         }
     }
 
+    /**
+     * Called when a local handler is unregistered, to send a `handler_unregistered`
+     * message to the far end.
+     */
+    public unregisterHandler(slotName: string, handler: Handler<any, any>): void {
+        if (this._localHandlers[slotName]) {
+            const ix = this._localHandlers[slotName].indexOf(handler)
+            if (ix > -1) {
+                this._localHandlers[slotName].splice(ix)
+                const unregistrationMessage: TransportUnregistrationMessage = {
```

Comment:
@lguychard we just realized with @highfivedenis that sending this message would unregister *all* the handlers registered on this specific slotname, and not only the one we wanted to remove.
A solution could be to add a unique ID to identify each remote handler at registration. What do you think?

Reply:
we took another direction: we only send `handler_registered` when a first handler is registered, and `handler_unregsitered` when the last handler is unregistered.